src/components/helpers/pretty-text-input.js

```diff
@@ -409,11 +402,16 @@ export default createReactClass({
 
     const textBox = this.textBoxRef;
     textBox.innerHTML = ''; // release any previous read-only content so it can be GC'ed
-    const pos = this.codeMirror ? this.codeMirror.getCursor() : null;
+
     this.codeMirror = this.props.config.codeMirror()(textBox, options);
     this.codeMirror.on('change', this.onCodeMirrorChange);
     this.codeMirror.on('focus', this.onFocusCodeMirror);
 
+    this.debouncedOnChangeAndTagCodeMirror = _.debounce(() => {
+      this.onChangeAndTagCodeMirror();
+    }, 200);
```

Comment:
Create the debounced version of `onChangeAndTagCodeMirror` when we create the editor instead of doing it on the fly in `onCodeMirrorChange`. This is mainly to better reason about setup/teardown code related to CM.